src/components/contextMenu/AposContextMenuItem.vue

```diff
@@ -12,45 +12,48 @@
 
 <script>
 export default {
-  name: 'AposContextMenuItem',
+  name: "AposContextMenuItem",
   props: {
-    menuItem: Object,
+    menuItem: {
+      type: Object,
+      default: null
+    },
     open: Boolean
   },
   computed: {
     tabindex() {
-      return this.open ? '0' : '-1';
+      return this.open ? "0" : "-1";
     }
   },
   methods: {
     click() {
-      this.$emit('clicked', this.menuItem.action);
+      this.$emit("clicked", this.menuItem.action);
     }
   }
-}
+};
 </script>
 
 <style lang="scss">
-  .apos-context-menu__button {
-    display: inline-block;
-    width: 100%;
-    padding: 10px 20px;
-    border: none;
-    text-align: left;
-    font-size: map-get($font-sizes, default);
-    letter-spacing: 0.5px;
+.apos-context-menu__button {
```

Comment:
These seem substantial. Was this broken or is there a ticket?

Reply:
I'm working on https://apostrophecms.atlassian.net/jira/software/projects/A30U/boards/4/backlog?selectedIssue=A30U-184. 

With the goal of making a package that can be imported into Apostrophe.

When started storybook, all these linter errors happened. As not all of us have really worked on these components, we should have some tests for the more complex pieces so that when we integrate we're sure the components behave as we expect and saves us headaches if some of these need to be refactored in any way.

Reply:
Okay. This seems to actually change styles, though. Unclear why that changed.

Reply:
It appears my editor got a little aggressive trying to "fix" things. Not sure why it's not following our rules in this repository... Would be nice though if we could use prettier or run eslint fix on a precommit hook or something so we don't need to think about formatting issues like this.

Reply:
Since this project was not linted at all before, I disabled auto fix in VS Code for the project to make sure I know what I was changing. Running eslint fix on precommit seems alright. No requirement to have prettier installed locally. It seems a little weird to not see what you're  committing before you do it, but should be fine.

Reply:
Ok. So do you want me to roll back all the formatting stuff and just commit the testing framework and make another PR for linting? It probably should play by the rules, right?

Reply:
That might be easier. Keeping the PR focused.